README.MD

```diff
@@ -37,9 +37,9 @@ npm install --no-optional
 ```
 .env.example > .env
 ```
-3. Rename **config.json.example** become **config.json** then fill all available values too
+3. Rename **config.example.example** become **config.json** then fill all available values too
```

Comment:
Instead of `config.example.example`, shouldn't it be `config.example.json`?
```suggestion
3. Rename **config.example.json** become **config.json** then fill all available values too
```